codewars/trailing_zeros.rb

```diff
@@ -0,0 +1,21 @@
+# Approach:
+# All factorial numbers could be simplified to prime numbers, and to calculate number of trailing zeros, it is
+# needed to calculate how many times 5 (all numbers that ends wth 0 could be exactly divided by 5),
+# used in simplified factorial
+
+def zeros(n)
+  if n < 0
+    puts "Can't find trailing zeros for a negative number"
```

Comment:
static values should be in constants
```
ERROR_TEXT = "Can't find trailing zeros for a negative number".freeze
```
and then
```
puts ERROR_TEXT  && return if n < 0
```
one string is more preferable